Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,26 @@
 
 ## Exercise I
 
-a)
+a) O(n) because it will only be as big as a.
 
 
-b)
+b) O(n^2) because of the nested loops and is reliant on size of n.
```

Comment:
Double check this one. It's a little tricky. Yes we have two loops, but you have to consider both loops individually. The outer and the inner loop are not incrementing at the same rate. . .